api/src/main/resources/messages.properties

```diff
@@ -104,4 +104,45 @@ ${project.parent.artifactId}.report.emergency.boys=Boys
 ${project.parent.artifactId}.report.emergency.girls=Girls
 ${project.parent.artifactId}.report.emergency.0_14years=(0-14 years)
 ${project.parent.artifactId}.report.emergency.above14years=(15 years and above)
-${project.parent.artifactId}.report.emergency.medicalAndSurgicalEmergency=Medical and surgical emergency
\ No newline at end of file
+${project.parent.artifactId}.report.emergency.medicalAndSurgicalEmergency=Medical and surgical emergency
+${project.parent.artifactId}.report.antenatal.firstVisit.label = First Visit
+${project.parent.artifactId}.report.antenatal.secondVisit.label = Second Visit
+${project.parent.artifactId}.report.antenatal.thirdVisit.label = Third Visit
+${project.parent.artifactId}.report.antenatal.fourthVisit.label = Fourth Visit
+${project.parent.artifactId}.report.antenatal.fifthVisit.label = Fifth Visit plus
+${project.parent.artifactId}.report.antenatal.total.label = Total
+${project.parent.artifactId}.report.antenatal.reportName=Antenatal
+${project.parent.artifactId}.report.antenatal.reportDescription=MSPP (Antenatal) report required by Haitian MoH to any clinic in Haiti
+${project.parent.artifactId}.report.antenatal.gestationWeeks = weeks of pregnancy
```

Comment:
`...  =  ...` should should look like `...=...` surely.

Reply:
Fixed